javascript/javascript1/week3/noteApp.js

```diff
@@ -0,0 +1,57 @@
+// NOTE TAKING APP
+
+// Save a note
+const notes = [];
+
+function saveNote(content, id) {
+    notes.push({content, id});
```

Comment:
You should check here, before pushing if 
```
content parameter is a string
id is a number
```

Reply:
Ok.